src/main/java/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.java

```diff
@@ -132,6 +139,27 @@ public RESTZaakafhandelParameters readZaakafhandelParameters(@PathParam("zaaktyp
         return zaakafhandelParametersConverter.convertZaakafhandelParameters(zaakafhandelParameters, true);
     }
 
+    /**
+     * Retrieve the referentietabelwaarden for a VELD of a FORMULIER for a ZAAKTYPE
+     *
+     * @return list of values or empty list if not found
+     */
+    @GET
+    @Path("{zaaktypeUUID}/{formulierDefinitie}/{veldDefinitie}")
+    public List<RESTReferentieTabelWaarde> findZaakafhandelParametersReferentieTabelWaarden(
```

Comment:
Maak de naam van deze methode gelijk aan die van de nieuwe methode in de typescript service ZaakafhandelParametersService. Zo zijn ze makkelijker terug te vinden.